sdk/core/Azure.Core.Experimental/src/DynamicData/DynamicData.cs

```diff
@@ -44,14 +44,23 @@ internal static JsonSerializerOptions GetSerializerOptions(DynamicDataOptions op
         {
             JsonSerializerOptions serializer = new()
```

Comment:
Nit: only because this threw me a little in lines 56 and 57 below, but could we call this `serializationOptions` or just `options`? I knew the properties you were setting as belonging to `JsonSerializerOptions`, but when I saw them on `serializer` I had to look if that was some abstraction we had.

Reply:
Sure, yeah, I just wanted something short and options was already taken.  I can change it.